hsts/handler.go

```diff
@@ -0,0 +1,87 @@
+package hsts
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	middleware "github.com/SKF/go-enlight-middleware"
+)
+
+const (
+	Header        string        = "Strict-Transport-Security"
+	DefaultMaxAge time.Duration = 365 * 24 * time.Hour
+)
+
+type Middleware struct {
+	Tracer middleware.Tracer
+
+	maxAge            time.Duration
+	includeSubDomains bool
+	preload           bool
+
+	cachedPolicy string
```

Comment:
Indentation. It's minor

Reply:
It is intentional. The `maxAge`, `includeSubDomains`, `preload` are grouped together since they are "settings". The cache is there for another reason.